.github/workflows/test.yml

```diff
@@ -25,21 +29,26 @@ jobs:
 
     steps:
       - name: Cancel previous runs
-        uses: styfle/cancel-workflow-action@0.6.0
+        uses: styfle/cancel-workflow-action@0.8.0
         with:
           access_token: ${{ github.token }}
 
-      - name: Checkout repository
+      - name: Checkout the repository
         uses: actions/checkout@v2
         with:
           ref: ${{ github.head_ref }}
 
-      - name: Setup Elixir
-        uses: erlef/setup-beam@v1
+      - name: Set up Elixir
+        uses: erlef/setup-elixir@v1
         with:
           otp-version: ${{ env.OTP_VERSION }}
           elixir-version: ${{ env.ELIXIR_VERSION }}
 
+      - name: Setup Node
```

Comment:
Maybe fixing small wording here to make it consistent - `Set up` and `Setup`

Reply:
Thanks for catching it. Fixed in 111d23f